### ISSUE TITLE
low accuracy with incorrect one-hot encoding in lstm_seq2seq.py example

### ISSUE THREAD

**Issue by tykimos** [keras-team/keras#13266]
I tested the example of "keras/examples/lstm_seq2seq.py" with "accuracy" metrics. As a result that the model didn't achieve high accuracy. I found that one-hot encoding after tokenizing wasn't applied for the back padding of sentences.

**Comment by gowthamkpr** [comment]
@tykimos Can you please send me a github gist of this issue and also what backend are you using and the version of keras. Thanks!

**Comment by tykimos** [comment]
@gowthamkpr 
- Gist Link is https://gist.github.com/tykimos/a082d8b78e49acf38e65956f927c996b
- Keras version is 2.2.5.

I write PR #13269 "#13266 Update lstm_seq2seq.py(from 22% to 87% acc)". Would you like to check it too?
